README.md

```diff
@@ -8,15 +8,15 @@ For questions and discussion join the [Keras Slack](https://keras-slack-autojoin
 ## Setup
 
 1. Clone this repository
-1. `pip install .`
-1. `python setup.py build_ext --inplace`
+2. `pip install .`
```

Comment:
Just an FYI, this change isn't really needed, markdown will automatically change the numbers so that they are increasing. So keeping it all as 1s make it easier to add new elements to the list.

Reply:
Thanks for making me aware, I didn't know that. I'll fix this.